src/Kiota.Builder/Plugins/PluginsGenerationService.cs

```diff
@@ -183,7 +183,7 @@ private static OpenApiDocument InlineRequestBodyAllOf(OpenApiDocument openApiDoc
                 {
                     foreach (var property in apiSchema.Properties)
                     {
-                        newSchema.Properties.Add(property.Key, property.Value);
+                        IDictionaryExtensions.TryAdd(newSchema.Properties, property.Key, property.Value);
```

Comment:
```suggestion
                        newSchema.Properties.TryAdd(property.Key, property.Value);
```

Reply:
There's ambiguity error raised in using `TryAdd` in the above context. Ambiguity from the below 2 extensions both imported in the file
`Microsoft.Kiota.Abstractions.Extensions.IDictionaryExtensions.TryAdd`
and 
`System.Collections.Generic.CollectionExtensions.TryAdd`

Are you suggesting we use the system one? Or just ignore the pre-compile checker warning?

Reply:
`IDictionaryExtensions.TryAdd` exists for older runtimes that do not have method in the Dictionary class. We should try use the method from the dotnet library to leverage any fixes/optimizations as Kiota is built with the latest sdk version.

Reply:
I'll check to verify why it's imported.

Reply:
The method `apiManifest.ApiDependencies.AddOrReplace` is from 'Microsoft.Kiota.Abstractions.Extensions' hence the importation.